bamoe-maven-repository-zip/maven-repository-zip-assembly.xml

```diff
@@ -9,13 +9,12 @@
       <directory>${m2.repo.location}</directory>
       <outputDirectory>/</outputDirectory>
       <includes>
-        <include>**/bamoe/**</include>
-        <include>**/drools/**</include>
-        <include>**/kie/**</include>
+        <include>**</include>
       </includes>
       <excludes>
         <exclude>**/_remote.repositories</exclude>
-        <exclude>**/maven-metadata-local.xml</exclude>
+        <exclude>**/maven-metadata**</exclude>
```

Comment:
Hi @baldimir. This new configuration is pretty comprehensive, and it may contain also hidden directories (e.g. .ci, .git, etc). I did not test it, so I may be wrong.
If the actual content of the zip is exactly what is required, it is fine for me.

Reply:
It is being executed over a Maven repository, like .m2. So this configuration basically describes what should be included and excluded from .m2 folder. Such folder shouldn't contain .git or any other similar folder.